uint8 Packetheader
uint8 ValidData
uint8 UltraF
uint8 UltraB
float32 EncL
float32 EncR
uint8 Infra0
uint8 Infra1
uint8 Infra2
uint8 Infra3
uint8 Infra4
uint8 Infra5

